docs/parcels.md

```diff
@@ -0,0 +1,110 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
+by an application, parcels use similar methodology as applications, and can be considered an advanced single spa feature so we recommend [starting with applications.](/docs/applications.md)
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Typically we recommend you mount a parcel within 
+the context of an application because the parcel will be unmounted with the application.
+
+## Parcel Lifecycles
```

Comment:
I think that before we dive into the lifecycles, it would be helpful to talk about how to create a parcel and how to use one. Let's explain that a parcel is just an object with 3-4 functions on it. And that the code that uses it does so by calling `singleSpa.mountRootParcel()` or an application prop's `mountParcel()` function

Reply:
I also think that we need to shepherd people down the path of using the same single spa helper libraries that are used for applications.

Our selling point to users is that they can take an existing react/angular component and easily "parcelify" it with a helper library so that it can be used by code in a different framework. I think we could do a better job of helping the reader understand that.